src/components/tooltip/tooltip.d.ts

```diff
@@ -19,6 +20,7 @@ export interface TooltipProps {
     inputSize?: 'small' | 'medium' | 'large';
     bgColor?: string;
     fontColor?: string;
```

Comment:
It's good to have the prop description comment in the ts file as well (I know some haven't been added here) but I imagine @grabkowski is going through all of the prop definitions and adding them

Reply:
Yeah I left them blank for that reason but happy to add them in if Kuba wants me to

Reply:
My thinking was hopefully someday we'll convert to TypeScript and only have prop definitions in one place, and if we add the comments as we're going along it'll mean someone doesn't have to do it further down the line

Not going to block the PR if you don't do it here though!

Reply:
Yeah agree, I'll keep a n eye on them and add any missing descriptions after Kuna's finished